provisioning/install-aste.sh

```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/env bash
+set -ex
+
+# Install the C++ vtk library
+sudo apt-get -y install libvtk7-dev
+sudo apt-get -y install libmetis-dev
+
+python3 -m pip install sympy scipy jinja2
+
+# Get aste
+if [ ! -d "aste/" ]; then
+    git clone --depth=1 --branch master https://github.com/precice/aste.git
+fi
+(
+    cd aste
+    git pull
+    mkdir -p build && cd build
+    cmake .. && make -j "$(nproc)"
+)
+
+# Add aste to PATH and libmetis to the library path
+echo "export PATH=\"\${HOME}/aste/build:\${PATH}\"" >>~/.bashrc
+echo "export LD_LIBRARY_PATH=\"\${HOME}/aste/build:\${LD_LIBRARY_PATH}\"" >>~/.bashrc
```

Comment:
In principle, we could also use `make install` to install these executables system-wide.

Reply:
True. The reason I avoid that (and that I use repositories) is to let people easily `git pull` newer versions, if needed. I find that `make install` on a non-standard prefix (especially not via a package manager) makes this confusing, as sometimes users think that they are using one version, while they are using another.